frontend/src/metabase/home/components/Activity.jsx

```diff
@@ -152,7 +152,9 @@ export default class Activity extends Component {
         if (item.table) {
           description.summary = (
             <span>
-              {t`saved a question about `}
+              {item.model === "dataset"
+                ? t`saved a dataset about `
```

Comment:
Should it be something like "saved a dataset based on Orders" instead of "saved a dataset about Orders"?

Reply:
Good call @alxnddr, fixed 👌